src/pages/index.tsx

```diff
@@ -138,8 +193,48 @@ const IndexPage: NextPage = () => {
         <ClientContent />
       ) : (
         <>
-          <Text marginBottom={4} as={"p"}>Install this app in your Dashboard and check extra powers!</Text>
-          {mounted && !global.location.href.includes("localhost") && <AddToSaleorForm />}
+          {isLocalHost && (
+            <Box>
+              <Text variant={"heading"} marginTop={8} as={"h2"}>
+                Installation
+              </Text>
+              <Text as={"p"} variant="body" marginTop={2}>
+                To install the app in your Saleor Dashboard, follow these steps:
```

Comment:
It doesnt fit well if below we have this `<form>`. I mean its a different of doing the same field.

So if we give manual steps of installation, ensure its `.... OR ....`